### DISCUSSION TITLE
author-tools checkers

### DISCUSSION THREAD

**Discussion by ajeanmahoney** [ietf-tools/editor#11] [General]
Should we repackage author-tools checkers into the editor, or should the editor access the checkers on author-tools via an API?

**Comment by rjsparks** [comment]
Is working offline a strict requirement or a nice to have?

**Comment by ajeanmahoney** [comment]
Given that the current setup requires that the editors ssh into a server to run the various checker scripts (that is, they must be online), the ability to work offline is a nice-to-have feature that would be helpful for those RPC staff members who travel to IETF meetings.
